src/Configuration/Definition.php

```diff
@@ -36,6 +36,10 @@ public function getConfigTreeBuilder()
                     ->info('Timezone used to calculate task run date')
                 ->end()
 
+                ->scalarNode('timezone_log')
```

Comment:
Why not `booleanNode` with default `false` value?

Reply:
done:https://github.com/lavary/crunz/pull/178/commits/4470a8aa42da207a05cce5e873d5480cf7f0be8f